### ISSUE TITLE
List component :: Search isn't triggered after typing

### ISSUE THREAD

**Issue by gskyarts** [mittwald/flow#903]
Hi,

I've notice the search functionality of the list component doesn't work as i expected. Currently, the search isn't triggered automatically after typing. Instead, users need to press Enter to confirm the input and initiate the search. Would it be possible to implement an auto-trigger for the search? A live example, what i mean is the navigation search on the document page.
